src/modules/ModuleStatementHome.tsx

```diff
@@ -0,0 +1,83 @@
+import { AnimatableNumber } from "@/components/core/AnimatableNumber";
+import { RichText } from "@/components/core/RichText";
+import { Section } from "@/components/core/Section";
+import { AssetFragment } from "@/lib/contentful/fragments/AssetFragment";
+import { ComponentStatisticFragment } from "@/lib/contentful/fragments/ComponentStatisticFragment";
+import { GetModuleStatementHomeById } from "@/lib/contentful/query/GetModuleStatementHomeById";
+import { cx } from "cva";
+import { readFragment, ResultOf } from "gql.tada";
+
+export const ModuleStatementHome = ({
+  data,
+}: {
+  data: ResultOf<typeof GetModuleStatementHomeById>;
+}) => (
+  <Section className="dark py-16 dsk:pt-32 dsk:pb-20 flex flex-col items-center">
```

Comment:
```suggestion
  <Section data-testid="ModuleStatementHome" className="dark py-16 dsk:pt-32 dsk:pb-20 flex flex-col items-center">
```

Reply:
Done, and I made `data-testid` a mandatory property on the `Section` component so that we (I) don't miss it in the future 👍 (but you can still forcibly pass `data-testid={undefined}` if you _really_ don't need it.